.github/workflows/rubucop.yml

```diff
@@ -22,11 +22,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby 2.6
-      uses: actions/setup-ruby@v1
+      uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.6.x
+        ruby-version: 2.6
+        bundler-cache: true
     - name: Rubocop
-      run: |
-        gem install bundler
-        bundle install --jobs 4 --retry 3
-        bundle exec rubocop -D -S
+      run: bundle exec rubocop -D -S -a
```

Comment:
I don't think autocorrect makes sense for the CI. The CI will pass, even if there are issues in the checked in code as long as they are fixable

Reply:
Oh yeah! Good catch!

Reply:
Good side about my making dumb mistakes like this — gem caching really works!

Reply:
I hadn't seen that option before, so now I'm going to go update a few workflows for that cache seeing 😄